serverless/serverless-metering.adoc

```diff
@@ -0,0 +1,71 @@
+[id="metering-serverless"]
+= Using metering with {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: metering-serverless
+
+As an {product-title} cluster administrator, you can use metering to analyze what is happening in your {ServerlessProductName} cluster.
+
+For more information about metering on {product-title}, see xref:../metering/metering-about-metering.adoc[About metering].
+
+[IMPORTANT]
+====
+Some of the linked instructions,such as installation instructions for metering on {product-title}, use the `openshift-metering` project and namespace. To use metering with {ServerlessProductName}, use `metering` as the project and namespace.
+====
+
+[id="installing-metering-serverless_{context}"]
+== Installing metering
+For information about installing metering on {product-title}, see xref:../metering/metering-installing-metering.adoc[Installing Metering].
+
+include::modules/serverless-metering-datasources.adoc[leveloffset=+1]
+include::modules/serverless-metering-queries.adoc[leveloffset=+1]
+
+[id="reports-metering-serverless_{context}"]
+== Running a metering report for Knative Serving
+You can run metering reports against Knative Serving by creating `Report` resources.
+You can find example reports located in the `reports` directory.
+Before you run a report, you must modify the input parameter within the `Report` resource to specify the start and end dates of the reporting period.
+
+Once you have provided the input parameters, you can run the report using the command:
+----
+$ oc apply -f reports/<report-name>.yml
+----
+You can then check the report as shown in the following example:
+----
+$ kubectl get report
+
+NAME                        QUERY                       SCHEDULE   RUNNING    FAILED   LAST REPORT TIME       AGE
+knative-service-cpu-usage   knative-service-cpu-usage              Finished            2019-06-30T23:59:59Z   10h
+----
+
+[id="reports-accessing-serverless_{context}"]
+== Accessing a metering report for Knative Serving
```

Comment:
I'm not sure this should be in our docs. Explaining to users how to get to certain reports isn't the job of our docs I think.